website/content/docs/connect/gateways/mesh-gateway/wan-federation-via-mesh-gateways.mdx

```diff
@@ -38,6 +38,8 @@ Operators looking to simplify their WAN deployment and minimize the exposed
 security surface area can elect to join these datacenters together using [mesh
 gateways](/docs/connect/gateways/mesh-gateway) to do so.
 
+![Mesh Gateway Architecture](/img/mesh-gateways.png)
```

Comment:
Added the new diagram to the WAN federation page, since it now shows both control plane and data plane communication models.